torch/csrc/inductor/aoti_eager/kernel_meta_info.cpp

```diff
@@ -92,30 +92,6 @@ bool TensorMetadata::operator==(const TensorMetadata& other) const {
   }
 }
 
-std::ostream& operator<<(
```

Comment:
Useful to have for debugging, is it not externally linked?

Reply:
No externally linked.